.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: "nuget"
+    directory: "/source/"
+    schedule:
+      interval: "daily"
```

Comment:
Should we also add a `time` property here? I'm thinking it might be valuable when we configure this to notify us via slack in the morning/start of the work day etc, rather than, say like lunchtime?

>By default, Dependabot checks for new versions at a random set time for the repository.

Reply:
Good suggestion, I'll do that.

Reply:
I've pushed a change to get Dependabot to run at 9am NZ, and I've added a GitHub subscription in our team channel for PRs with the `dependencies` label

Reply:
Nice. I'm happy either way, not strongly held to any of the approaches mentioned.